.gitlab-ci.yml

```diff
@@ -35,15 +36,15 @@ RPM:
           - aws/fedora-32-x86_64
           - aws/fedora-33-x86_64
           - aws/fedora-33-aarch64
-          - aws/fedora-34-x86_64
-          - aws/fedora-34-aarch64
+#          - aws/fedora-34-x86_64
+#          - aws/fedora-34-aarch64
           - aws/rhel-8-x86_64
           - aws/rhel-8-aarch64
           - aws/centos-stream-8-x86_64
           - aws/centos-stream-8-aarch64
-      - RUNNER:
-          - aws/rhel-8.4-x86_64
-        INTERNAL_NETWORK: ["true"]
+#      - RUNNER:
+#          - aws/rhel-8.4-x86_64
+#        INTERNAL_NETWORK: ["true"]
```

Comment:
What is the actual problem that you are getting here ?

Reply:
The missing 8.4 latest nightly symlink... https://gitlab.com/osbuild/ci/osbuild-composer/-/jobs/1243772878 But I can see they brought it back now.. I need to ask someone what's happening.